static/style.css

```diff
@@ -108,6 +115,25 @@ textarea {
     padding-left: .25em;
 }
 
+.overlay {
+    background: rgba(0,0,0,.7);
+    width: 80%;
+    max-width: 900px;
+    max-height: 400px;
+    height: 80%;
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    margin-left: -25%;
+    display: none;
+    text-align: center;
+}
+
```

Comment:
This looks a bit strange on my screen, but we're more concerned with functionality than we are with layout.
![screen shot 2015-07-14 at 4 02 38 pm](https://cloud.githubusercontent.com/assets/3937302/8687131/eaa9777e-2a41-11e5-8275-43788a631a50.png)